assets/src/dashboard/animations/configs/index.js

```diff
@@ -17,17 +17,19 @@
 /**
  * Internal dependencies
  */
-import { ANIMATION_TYPE } from '../constants';
+import { ANIMATION_TYPES } from '../constants';
```

Comment:
Updated it to plural name to be consistent with other constants in dashboard project.